bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -56,12 +57,49 @@ spec:
               exit 0
           }
           trap quit SIGTERM
-          /usr/share/openvswitch/scripts/ovs-ctl start --ovs-user=openvswitch:openvswitch --system-id=random
-          ovs-appctl vlog/set "file:${OVS_LOG_LEVEL}"
-          /usr/share/openvswitch/scripts/ovs-ctl --protocol=udp --dport=6081 enable-protocol
+          chroot /host /bin/bash <<EOF
```

Comment:
I think you likely want to use [quoted EOF](https://stackoverflow.com/questions/22697688/how-to-cat-eof-a-file-containing-code), otherwise e.g. I bet the `$?` is being expanded to `0` from the outer shell's last command success status...